src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/1.sql

```diff
@@ -193,8 +193,47 @@ CREATE UNIQUE NONCLUSTERED INDEX IX_Resource_ResourceTypeId_ResourceId ON dbo.Re
 INCLUDE (Version)
 WHERE IsHistory = 0
 
+/*************************************************************
+    Capture claims on write
+**************************************************************/
+
+CREATE TABLE dbo.ClaimType
+(
+    ClaimTypeId tinyint IDENTITY(1,1) NOT NULL,
+    Name varchar(128) NOT NULL
+)
+
+CREATE UNIQUE CLUSTERED INDEX IXC_Claim on dbo.ClaimType
+(
+    Name
+)
+
+CREATE TYPE dbo.ResourceWriteClaimTableType AS TABLE  
+(
+    ClaimId tinyint NOT NULL,
```

Comment:
Does ClaimId map to ClaimTypeId?  If so, any reason for the different names?

Reply:
Oh yeah, that's supposed to be ClaimTypeId.